views/layouts/MainLayout.jsx

```diff
@@ -50,68 +33,53 @@ const MainLayout = ({ classes, theme }) => {
       setNotification({ open: true, message: error.message })
     }
   }
-  const handleDashboardContent = () => {
-    if (pathname.includes(dashboard)) {
-      setDrawerVariant(TEMPORARY_SIDEBAR)
-    } else {
-      setDrawerVariant(PERSISTENT_SIDEBAR)
-      setOpenSidebar(true)
-    }
-  }
-  const fetchCounts = async () => {
-    const { numOfSites, maxDay, numOfParticipants } = await api.counts.all()
+  const handleLogout = async () => {
+    try {
+      await api.auth.logout()
 
-    setSideBarState({
-      totalDays: maxDay,
-      totalStudies: numOfSites,
-      totalSubjects: numOfParticipants,
-    })
+      setUser(null)
+      navigate(routes.login)
+    } catch (error) {
+      alert(error.message)
+    }
   }
-
   useEffect(() => {
-    fetchCounts()
     fetchUsers()
     loadAllConfigurations()
   }, [])
-  useEffect(() => {
-    handleDashboardContent()
-  }, [pathname])
 
   return (
-    <div className={classes.root}>
-      <Header
-        onToggleSidebar={toggleSidebar}
-        title={headerTitle(pathname, params)}
-        isAccountPage={false}
-        user={user}
-      />
-      <Sidebar
-        drawerVariant={drawerVariant}
-        onToggleSidebar={toggleSidebar}
-        sidebarOpen={openSidebar}
-        totalDays={sideBarState.totalDays}
-        totalStudies={sideBarState.totalStudies}
-        totalSubjects={sideBarState.totalSubjects}
-      />
-      <div className={classNames(classes.content, classes.contentPadded)}>
+    <Box sx={{ display: 'flex', flexDirection: 'row' }}>
+      <Box
+        component="nav"
+        sx={{
+          width: 248,
```

Comment:
if the width is defined here it shouldn't have to be defined in the `Sidebar` component.

Reply:
[465c369](https://github.com/AMP-SCZ/dpdash/pull/458/commits/465c36952e0f8176957c65d89dfea1c6b86ca296)